tests/rewriter_test.py

```diff
@@ -194,3 +194,79 @@ def transformation(module: ModuleOp, rewriter: Rewriter) -> None:
         rewriter.inline_block_after(if_block, constant_op)
 
     rewrite_and_compare(prog, expected, transformation)
+
+
+def test_insert_block():
+    """Test the insertion of a block in a region."""
+    prog = \
+    """module() {
+  %0 : !i1 = arith.constant() ["value" = 1 : !i1]
+}"""
+
+    expected = \
+"""module() {
+^0: ^1: 
```

Comment:
Should an empty block be on the same line as the next block? I think the following would be nicer, since it highlights more clearly that there are two blocks.

```
^0:
^1: 
   ....
```